src/core/index.ts

```diff
@@ -75,16 +96,39 @@ interface BellPromptConfig {
     };
     colors?: {
         bell?: {
+            fg?: string;
             bg?: string;
+        };
+    };
+}
+
+export interface AlertPromptConfig extends BasePromptConfig {
+    type: 'alert';
+    trigger: PromptTrigger;
+    position: 'top-center';
+    labels?: {
+        thanksForSubscribing?: string;
+        alert?: {
+            heading?: string;
+            body?: string;
+            declineAction?: string;
+            acceptAction?: string;
+        };
+    };
+    colors?: {
+        alert?: {
             fg?: string;
+            bg?: string;
+            declineActionFg?: string;
+            declineActionBg?: string;
+            acceptActionFg?: string;
+            acceptActionBg?: string;
         };
     };
-    position: 'bottom-left' | 'bottom-right';
-    overlay?: PromptOverlayConfig;
-    actions?: PromptAction[];
+    iconUrl?: string;
```

Comment:
Can/should icon come from the web push site configuration, similar to the Overlay shown whilst requesting native permission?

Reply:
Yes, now done 👍